### ISSUE TITLE
fastforward and rewind broken in ps4 browser

### ISSUE THREAD

**Issue by Djang007** [iv-org/invidious#712]
Sometimes when I'm away from my PC I like to use invidious on my PS4 to watch HD videos.  

I use the PS4's standard browser because the official YouTube app sucks and invidious lets me watch region locked videos.  

Lately I've noticed that something has changed and I can no longer use the D-pad on the PS4 controller to fast-forward and rewind videos whenever I watch them in the ps4 browser. Is anyone else getting the same problem on their ps4?

**Comment by omarroth** [comment]
Would you mind testing on https://dev.invidio.us to confirm that #713 fixes the issue for you?

**Comment by Djang007** [comment]
Sorry for the late reply, I've been busy. I checked to see if the fix worked at https://dev.invidio.us and the issue still persist. 

I did notice however that I could use the D-pad controls when manually clicking on the play bar icon. Whatever is causing the bug it seems to happen when going into full screen mode.  Can anyone with a ps4 see if they have the same issue? That way we would be able to tell if it's a bug or something on my end causing the issue.
